docs/conf.py

```diff
@@ -97,7 +97,7 @@
 # The short X.Y version.
 version = "0.11.0"
 # The full version, including alpha/beta/rc tags.
-release = "0.11.0-beta.2"
+release = "0.11.0-rc1"
```

Comment:
use consistently a format compliant with Semver
```suggestion
release = "0.11.0-rc.1"
```